syntax = "proto3";
package http;
option cc_generic_services = true;

message HttpRequest {};
message HttpResponse {};

service HttpService {
    rpc HttpTalk(HttpRequest) returns (HttpResponse);
};
